pkg/api/openshiftcluster.go

```diff
@@ -169,14 +169,24 @@ func (t ProvisioningState) String() string {
 	return string(t)
 }
 
+// FipsValidiatedModules determines if FIPS is used.
+type FipsValidatedModules string
+
+// FipsValidatedModules constants.
+const (
+	FipsValidatedModulesEnabled  FipsValidatedModules = "Enabled"
+	FipsValidatedModulesDisabled FipsValidatedModules = "Disabled"
+)
+
 // ClusterProfile represents a cluster profile.
 type ClusterProfile struct {
 	MissingFields
 
-	PullSecret      SecureString `json:"pullSecret,omitempty"`
-	Domain          string       `json:"domain,omitempty"`
-	Version         string       `json:"version,omitempty"`
-	ResourceGroupID string       `json:"resourceGroupId,omitempty"`
+	PullSecret           SecureString `json:"pullSecret,omitempty"`
+	Domain               string       `json:"domain,omitempty"`
+	Version              string       `json:"version,omitempty"`
+	ResourceGroupID      string       `json:"resourceGroupId,omitempty"`
+	FipsValidatedModules string       `json:"fipsValidatedModules,omitempty"`
```

Comment:
This should be not a string type, but a `FipsValidatedModules` type.  Then you don't have to convert to a string everywhere.  See the SDN stuff for inspiration.

Reply:
oversight on my part. nice catch